packages/core/package.json

```diff
@@ -1,13 +1,20 @@
 {
-  "name": "@loopback/loopback",
+  "name": "@loopback/core",
   "version": "4.0.0-alpha.1",
   "description": "",
   "main": "index",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
+    "build": "npm run build:lib && npm run build:lib6",
+    "build:lib": "tsc --target es2017 --outDir lib",
+    "build:lib6": "tsc --target es2015 --outDir lib6",
+    "clean": "rm *.tgz; rm -rf ./package",
+    "prepublish": "npm run build",
+    "pretest": "npm run build",
     "integration": "mocha --opts ../../test/mocha.opts 'test/integration/**/*.ts'",
     "test": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts' 'test/integration/**/*.ts' 'test/acceptance/**/*.ts'",
-    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'"
+    "unit": "mocha --opts ../../test/mocha.opts 'test/unit/**/*.ts'",
+    "verify": "npm run clean && npm pack && tar xvzf *.tgz && tree package"
```

Comment:
Actually, it's a bit better if we just pipe to sort:

`tar tf *.tgz | sort`, so I can live with this one too

```
package/README.md
package/index.d.ts
package/index.js
package/lib/binding.d.ts
package/lib/binding.js
package/lib/context.d.ts
package/lib/context.js
package/lib/index.d.ts
package/lib/index.js
package/lib/inject.d.ts
package/lib/inject.js
package/lib/isPromise.d.ts
package/lib/isPromise.js
package/lib/resolver.d.ts
package/lib/resolver.js
package/lib6/binding.d.ts
package/lib6/binding.js
package/lib6/context.d.ts
package/lib6/context.js
package/lib6/index.d.ts
package/lib6/index.js
package/lib6/inject.d.ts
package/lib6/inject.js
package/lib6/isPromise.d.ts
package/lib6/isPromise.js
package/lib6/resolver.d.ts
package/lib6/resolver.js
package/package.json
```